examples/hotrod/cmd/frontend.go

```diff
@@ -45,7 +45,7 @@ var frontendCmd = &cobra.Command{
 		logger := log.NewFactory(zapLogger)
 		server := frontend.NewServer(
 			options,
-			tracing.Init("frontend", otelExporter, metricsFactory, logger),
+			tracing.InitOTEL("frontend", otelExporter, metricsFactory, logger),
```

Comment:
@yurishkuro in order to separate out a PR, we'll have to create a duplicate instance of tracer here to be initiated by NewServer constructor.

Reply:
Unfortunately that way didn't work out for me. We are loosing span from context.